Cargo.toml

```diff
@@ -10,16 +10,16 @@ serde = "1.0.53"
 serde_derive = "1.0.53"
 
 [dependencies.pyo3]
-version = "0.4.0"
+version = "0.5.0"
 features = ["extension-module"]
```

Comment:
That line shouldn't be here

Reply:
This happens when I code at night and cook noodles on the side:
I forget to commit half of my changes while in the meantime the kitchen almost burns down. :man_cook: :disappointed:

Reply:
And they told us doing things in parallel would be fine in rust :wink:  🍜

Reply:
BTW tweeted that: https://twitter.com/matthiasendler/status/1062838581914427393